File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PhotoFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.PhotoSize;

public class PhotoFilterValidator extends FilterValidator<PhotoSize> {

    public PhotoFilterValidator(CustomValidator<PhotoSize> validator) {
        super(PhotoSize.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/PhotoFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.std.files.PhotoSize
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class PhotoFilterValidator(validator: Validator<PhotoSize>) : FilterValidator<PhotoSize>(PhotoSize::class, validator)

